agent/util-scripts/pbench-tool-meister-start

```diff
@@ -420,6 +420,11 @@ class BaseServer:
                             self.bad_port_ret_code,
                         ) from exc
             host = host_port_parts[0] if host_port_parts[0] else def_host_name
+            if host[0] == "[" and host[-1] == "]":
+                # Brackets are invalid for a host name, but might be used when
+                # specifying a port with an IPv6 address, strip them to before
```

Comment:
Typo: "...strip them before..."

Although I would prefer to handle bracket consistency checks as part of the validation; `[pbench.example.com]` and `[10.0.0.1]` are not valid addresses, but `[fe00::0]` is ...

Still, as a hack to avoid collapsing with valid IPv6 addresses, this is a decent first step. ;-)

Reply:
Supporting brackets is weird.  The `ipaddress` module does not accept the brackets, and I am pretty sure that specifying a host name with brackets is not accepted for other interfaces.  So we have to strip the brackets no matter what.

And yes, IPv6 is supported in our lab infrastructure, and it turns out that we use it locally for communicating with our GraphQL instance.